dist/index.js

```diff
@@ -10364,16 +10364,25 @@ const run = async () => {
             process.exit(0);
         }
 
+        const [branchName, project] = branch.split('/');
```

Comment:
we shall consider cases when `release` has no project, like in Web Config or Web Shared

Reply:
I didn't realize that case 😕, how is the changelog action going to know the release branch is for x project?

Reply:
if `release` branch name contains no project, then it serves single project

Reply:
But this is the problem for the changelog action besides the title there is no indicative of which project is released, for example in this PR https://github.com/zattoo/web-shared/pull/1159

So it's quite hard to figure out which one of the changelogs is the important one that should be modified and ignore the others projects for asking changes; In the past I suggested asking for at least 1 changelog in release branches and ignore asking based on modified files, but we dismissed this solution because we can also have multiple released in a single PR.

Reply:
I suggest continuing with a soft validation,
if a branch doesn't include project e.g. `release`, we shall have **at least one** changelog entry that encapsulates a release, if non is modified fail.
I think we shall consider release branches special cases where we don't care too much about modified files (which shall be validated earlier on main) but rather on a release entry in the changelog, after all this is how we release packages.

Reply:
The check for a minimum of one changelog modified for release was already in place: https://github.com/zattoo/changelog/blob/master/src/index.js#L78 

I added the other check to not ask for modified files if a release does not have a project.

Reply:
@dobladov but it doesn't look like we're validating release, since there is a condition for checking only branches with project 
https://github.com/zattoo/changelog/blob/e7f898f40552f5b1fae400a1a7a49b1f3e7fbdc1/src/index.js#L82
correct me if I'm wrong

Reply:
We could validate here only if `changelogs` length is 1, I can add the condition it's a good point 👍 , we were ignoring this for releases without projects because it might contain more than one and then is not clear which one to validate